test/amqp-integration-test.py

```diff
@@ -222,14 +223,32 @@ def cleanup():
     else:
         print("\n\nFAILURE %d" % exit_status)
 
-
 exit_status = ExitStatus.OK
 tempdir = tempfile.mkdtemp()
+
+parser = argparse.ArgumentParser(description='Run integration tests')
+parser.add_argument('--no-node', dest="run_node", action="store_false",
```

Comment:
Yeah, I can do that! I can even go further and make an --all that runs all of them no matter what test flags we add later.